http/handler.go

```diff
@@ -177,6 +177,7 @@ func (h *Handler) populateValidators() {
 	h.validators["GetFragmentData"] = queryValidationSpecRequired("index", "field", "shard")
 	h.validators["PostFragmentData"] = queryValidationSpecRequired("index", "field", "shard")
 	h.validators["GetFragmentBlocks"] = queryValidationSpecRequired("index", "field", "view", "shard")
+	h.validators["ImportRoaringBitmap"] = queryValidationSpecRequired("index", "field", "shard")
```

Comment:
This validator is not set up correct. In fact, I thing there are a few here that are not correct.

Also, in the client the method is `ImportRoaringBytes`.